README.md

```diff
@@ -25,7 +25,7 @@ by adding `opentelemetry_tesla` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:opentelemetry_tesla, "~> 1.3.2-rc.1"}
+    {:opentelemetry_tesla, "~> 1.3.2-rc.3"}
```

Comment:
would be nice if we could get rid of `-rc` suffix, but I guess we cannot because opentelemetry_telemetry is still using beta version

Reply:
exactly, not sure how we are going to manage this when this library moves to https://github.com/open-telemetry/opentelemetry-erlang-contrib